microhapulator/data/template.html

```diff
@@ -165,6 +227,29 @@ <h2>Haplotype Calling</h2>
                 {% endfor %}
             </table>
 
+            Some explanatory text here.
+                <table id="typing_rates_table" class="pagination_table" >
+                    <thead>
+                    <tr>
+                        <th>Marker</th>
+                        {% for sample in typing_rates.keys() %}
+                        <th style="font-weight:bold">{{sample}}</th>
+                        {% endfor %}
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for markername in markernames %}
+                    <tr>
+                        <td><a href="marker-detail-report.html?marker={{markername}}" target="_blank">{{markername}}</a></td>
+                        {% for sample_data in typing_rates.values()%}
+                        <td class="alnrt">{{"{:,.2f}".format(sample_data.loc[markername, 'TypingRate']*100)}}%</td>
+                        {% endfor %}
+                    </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+
+        
             <a name="filters"></a>
             <h2>Genotype Calling</h2>
             <p>Fixed detection thresholds and dynamic analytical thresholds are applied to the typing result using <code>mhpl8r filter</code> to discriminate true and erroneous haplotypes and predict each sample's genotype. MicroHapulator applied a static filter of <strong>≥{{static}} reads</strong> as a detection threshold and a dynamic filter of <strong>≥{{"{:.1f}".format(dynamic*100)}}% of total reads</strong>.</p>
```

Comment:
This line includes a couple of `≥` characters. I know these were rendered correctly in my initial testing, but it looks like there's a text encoding issue and they're causing rendering artifacts. I don't think it has anything to do with the changes added in this PR. But if after a few minutes of troubleshooting we can't find an explanation or fix, we can just replace it with `>=`.

Reply:
The quick solution I found was to just use the hex code for ≥. It renders correctly now but it certainly less human readable when looking at the html template so I could change it to >= if you think that's better.

Reply:
Looks good.